docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,47 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-10-1]]
+=== Logstash 7.10.1 Release Notes
+
+==== Notable issues fixed
+
+===== Fix docker image metadata
+Removed non-OCI compliant freeform metadata labels, started using consistent build date for all the docker images and dockerfiles. https://github.com/elastic/logstash/pull/12451[#12451]
+Reintroduces the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
```

Comment:
Use consistent tense for release note comments

```suggestion
Reintroduced the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
```

Reply:
Thank's Rob, as Joao is suggesting, the message for this release is the fix of pipeline reloading other than the other https://github.com/elastic/logstash/pull/12489#discussion_r537550747 so probably this part is going to be removed